frb_codegen/src/generator/dart/ty_sync_return.rs

```diff
@@ -0,0 +1,46 @@
+use crate::generator::dart::ty::*;
+use crate::ir::*;
+use crate::type_dart_generator_struct;
+use crate::utils::BlockIndex;
+
+type_dart_generator_struct!(TypeSyncReturnGenerator, IrTypeSyncReturn);
+
+impl TypeDartGeneratorTrait for TypeSyncReturnGenerator<'_> {
+    fn api2wire_body(&self, _: BlockIndex) -> Option<String> {
+        Some("/*unsupported*/".to_string())
```

Comment:
uber-nit: Shall we directcly panic when this happens? e.g. `unimplemented!()`

Reply:
I was referring to the implementation of `SyncReturnVecU8` which just simply returns an "/*unsupported*/" for the wire function. I also think that letting it crash is a better idea since it's less confusing, and it will immediately reveal the problem if there's any.

Reply:
Sure, that old code was written by me (a year ago when I did not expect the lib will be so popular!) I agree with you - crash early would be better